packages/a11y/src/prefersReducedMotion.tsx

```diff
@@ -0,0 +1,21 @@
+import { css } from '@leafygreen-ui/emotion';
+
+/**
+ * Returns an emotion CSS instance for rendering styles that respect
+ * OS-level preferences for reduced motion.
+ *
+ * Use this function to remove scale, size, and positional transitions
+ * for users with that preference set.
+ * @param styles String of styles to render within media query.
+ */
+export default function prefersReducedMotion(styles: string) {
```

Comment:
This doesn't help us handle suppressing animations that exist in components when prefers-reduced-motion is set. Perhaps we can also add a hook that can be used in a similar way that our `useUsingKeyboardProvider` hook is used to enable us to toggle on/off animations and transitions

Reply:
Thinking even further about this, maybe we want to see about injecting global styles that respond to this media query

Reply:
I wanted to keep this solution to CSS-only.

On the global styles front, that's quite difficult. We could unset every CSS `transition` on the page, but the problem is that some transitions like opacity and color are appropriate even when the user has "reduce motion" set.